### ISSUE TITLE
How do I run the project (imposm, tegola)?

### ISSUE THREAD

**Issue by mifozski** [openinframap/openinframap#184]
I'm not very proficient with Docker and services so it would be nice to have a more detailed guide on how to deploy the backend services. 
Right now, I have the images built from the Dockerfiles in imposm and tegola folders but what do I with them afterwards?
Just running `docker run but it just gives me error:
```
Available commands:
        import
        diff
        run
        query-cache
        version
Usage: /usr/bin/imposm COMMAND [args]
```

**Comment by flacombe** [comment]
Hello

For imposm, the docker exposes directly its binary.
So arguments should be the ones used in the imposm documentation: https://imposm.org/docs/imposm3/latest/tutorial.html#importing

let's see if you have issues with other services afterwards

**Comment by russss** [comment]
I have added some more documentation [here](https://github.com/openinframap/openinframap/tree/main/imposm) recently, but it is a relatively complex system and it's easy to get stuck if you aren't familiar with all the component parts. I don't really have enough time to provide support on this myself, unfortunately.

**Comment by mifozski** [comment]
Thanks for adding the guidelines! I managed to follow them through and start the project with imported data.
The only thing I had to tweak was to add `ALTER ROLE osm SUPERUSER;` to `dev-init.sql` otherwise I had problems with installing the database extensions.
And also add `- ./schema/views.sql:/schema/views.sql` to the db volumes in `docker-compose.yml` so it's easier to access and add views to the db after the import.
